GitUI/CommandsDialogs/BrowseDialog/FormRecentReposSettings.Designer.cs

```diff
@@ -153,7 +156,7 @@ private void InitializeComponent()
             this.comboMinWidthNote.Location = new System.Drawing.Point(11, 246);
             this.comboMinWidthNote.Name = "comboMinWidthNote";
             this.comboMinWidthNote.Padding = new System.Windows.Forms.Padding(12, 0, 0, 0);
-            this.comboMinWidthNote.Size = new System.Drawing.Size(300, 73);
+            this.comboMinWidthNote.Size = new System.Drawing.Size(300, 30);
```

Comment:
Why?

Reply:
This magically fixed the UI, the combobox is never rendered with this height anyways.

With height 73:
![grafik](https://user-images.githubusercontent.com/7032094/77310712-13a0ff80-6cff-11ea-939f-4106f0275567.png)
Height 30:
![grafik](https://user-images.githubusercontent.com/7032094/77310737-1b60a400-6cff-11ea-8ce9-691bde4da4b1.png)